db/migrate/20160122052547_create_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+  	create_table :users do |t|
+  		t.string :username
+  		t.string :email
+  		t.string :password
```

Comment:
Should be `password_digest` for the `has_secure_password` functionality and pattern.